### PR TITLE
CI Tests: Add descriptive names to vrep shards. Update test generator script

### PR DESCRIPTION
**Pull request by rohit-nayak-ps**
* Added descriptive names for vreplication shards
* Moved v2 e2e test to its own shard 
I did this because it was failing often while trying to create a replica, presumably running out of resources. I thought it might be because there were other tests running. However I am still seeing this happening while CI ran for this PR.

### CODE REVIEW

File: .github/workflows/cluster_endtoend_11.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# DO NOT MODIFY: THIS FILE IS GENERATED USING test/ci_workflow_gen.go
```
Comment (deepthi) [inline .github/workflows/cluster_endtoend_11.yml:1]: What do you think about saying that these files are generated using `make generate_ci_workflows`?

**Review by deepthi — APPROVED** [review]
LGTM, just one comment.

File: test/ci_workflow_gen.go [hunk 2]
```diff
@@ -31,17 +31,16 @@ const (
...
+	clusterList         = "11,12,13,14,15,16,17,18,19,20,21,22,23,24,26,27,vreplication_basic,vreplication_multicell,vreplication_cellalias,vreplication_v2"
 	// TODO: currently some percona tools including xtrabackup are installed on all clusters, we can possibly optimize
 	// this by only installing them in the required clusters
 	clustersRequiringXtraBackup = clusterList
+	clustersRequiringMakeTools  = "18,24"
 )
 
 type unitTest struct {
-	Name, Database string
+	Name, Platform string
```
Comment (deepthi) [inline test/ci_workflow_gen.go:43]: I assume this change fixes the error we were getting from `make`:
```
Error: template: unit_test.tpl:22:17: executing "unit_test.tpl" at <.Platform>: can't evaluate field Platform in type *main.unitTest
```

File: .github/workflows/cluster_endtoend_11.yml (hunk 1 above)
Comment (rohit-nayak-ps) [reply .github/workflows/cluster_endtoend_11.yml:1]: Good idea, will make the change

File: test/ci_workflow_gen.go (hunk 2 above)
Comment (rohit-nayak-ps) [reply test/ci_workflow_gen.go:43]: Yes,

**Review by deepthi — APPROVED** [review]

**Comment by shlomi-noach** [comment]
Added tests as `Required`

Screenshot from 2021-02-07 08-53-42

**Pull request merged**